samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -187,3 +196,35 @@ HorizontalVirtualGridList.parameters = {
 export const WithButtonSpotlightGoesToCorrectTarget = () => <ButtonAndVirtualGridList />;
 
 WithButtonSpotlightGoesToCorrectTarget.storyName = 'with Button, Spotlight goes to correct target';
+
+export const HorizontalSquaredVirtualGridList = () => (
```

Comment:
I can see only one item in this sample
Did I something wrong?
![image](https://user-images.githubusercontent.com/34676178/117260357-f6d77500-ae89-11eb-9a5c-f220351e9c00.png)

Reply:
The horizontal scrollbar is hidden. You need to wheel or 5way right to go to the next item. =)